CI release fails

See https://github.com/ChainSafe/js-libp2p-gossipsub/runs/6731255681?check_suite_focus=true

```

npm notice === Tarball Details === 
npm notice name:          @chainsafe/libp2p-gossipsub             
npm notice version:       1.1.1                                   
npm notice filename:      @chainsafe/libp2p-gossipsub-1.1.1.tgz   
npm notice package size:  90.8 kB                                 
npm notice unpacked size: 437.8 kB                                
npm notice shasum:        a8e37c2d5df1227ef63ecdb160c961fb2fe9f384
npm notice integrity:     sha512-MO9DEKEtFt5TY[...]8d2c91KZKzqMw==
npm notice total files:   97                                      
npm notice 
npm notice Publishing to https://registry.npmjs.org/
npm ERR! code E404
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/@chainsafe%2flibp2p-gossipsub - Not found
npm ERR! 404 
npm ERR! 404  '@chainsafe/libp2p-gossipsub@1.1.1' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -100,3 +100,5 @@ jobs:
 
       - run: npm publish --access public 
         if: ${{ steps.release.outputs.release_created }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}

--- a/.github/workflows/semantic-pr.yml
+++ b/.github/workflows/semantic-pr.yml
@@ -20,5 +20,3 @@ jobs:
             fix
             feat
             chore
-          validateSingleCommit: true #single commit can ovveride squash merge commit message
-          validateSingleCommitMatchesPrTitle: false

